mantidproject/mantid
# Add threshold filter to the VSI standard view.

Description of work.

This adds a button that applies the vtkThreshold filter to the VSI standard view. It also updates the cut button to match the equivalent button in ParaView.

**To test:**

1. Load a MDHistoWorkspace in Mantid and open the VSI.
2. If not already on the standard view, switch to it.
3. Click the ![pqslice24](https://cloud.githubusercontent.com/assets/5920552/20945851/c9805716-bbd6-11e6-8983-55485a5333cb.png) button and ensure the slice functionality still works.
4. Click the ![pqthreshold24](https://cloud.githubusercontent.com/assets/5920552/20945852/c9860fb2-bbd6-11e6-8f95-38697ae24b7f.png) button and verify that the threshold filter is in the pipeline and changing the minimum and maximum values in the properties panel changes the output.



There is no GitHub issue associated with this pull request

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
